spec/feature/takeaway_feature_spec.rb

```diff
@@ -0,0 +1,31 @@
+
+describe 'Feature test' do
+
+  let(:name) {"Rufus"}
+  let(:tel_no) {"1-800-EXAMPLE"}
+  let(:message) {"Thank you for your custom. Your total is $11. Your order will be delivered at #{Time.now + 3600}"}
```

Comment:
Metrics/LineLength: Line is too long. [116/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)